app/models/component.rb

```diff
@@ -95,6 +95,10 @@ def display
     I18n.t("user_journey.component_name.#{type}")
   end
 
+  def display_long_name
+    I18n.t("user_journey.component_long_name.#{type}")
+  end
+
```

Comment:
Apologies if this is being extremely nit-picky...

It is clear what this method is doing, which is good; though it does make the 'display' on line 94 method look rather ambiguous in comparison. Maybe change 'display' to 'display_name' to be clear?

Reply:
I can do as part of the next PR as a refactor. Making it here would add another 26 changed lines and 10 files to this PR (more than this change itself)...

Reply:
ahhhh...never mind then 😄